lib/queue.js

```diff
@@ -257,7 +257,9 @@ Queue.prototype.distEmit = function(){
 
 Queue.prototype.on = function(){
   var args = Array.prototype.slice.call(arguments);
-  Disturbed.prototype.on.apply(this, args);
+  var promise = Disturbed.prototype.on.apply(this, args);
+  var _this = this;
+  promise.catch(function(err){ _this.emit('error', err); });
   return this;
```

Comment:
I changed the API in disturbed so that it will handle local events as well which will break this. I'll fix it asap.